cmake/GenericTargets.cmake

```diff
@@ -46,6 +46,7 @@ function (generate_generic_builtins ispc_name)
         "i1x8"
         "i1x16"
         "i1x32"
+        "i1x64"
```

Comment:
Please, remove the comment above reffering to #3123

Reply:
Done